spec/views/admin/submissions/show.html.erb_spec.rb

```diff
@@ -198,6 +196,37 @@
           expect(page).to have_content('Note 3')
         end
       end
+
+      it 'shows a notes section even if there are no notes' do
+        page.visit "/admin/submissions/#{@submission.id}"
+
```

Comment:
probably an unnecessary test - i wrote this initially because i was undoing a previous assertion above where i removed L50.

Reply:
Agreed that the test below would cover the case we want to test here.

Reply:
I agree - let's remove this test for now.